src/connector/src/parser/mysql.rs

```diff
@@ -149,7 +149,7 @@ mod tests {
     use mysql_async::Row as MySqlRow;
     use risingwave_common::catalog::{Field, Schema};
     use risingwave_common::row::Row;
-    use risingwave_common::types::{DataType, ToText};
+    use risingwave_common::types::DataType;
```

Comment:
This change was made during browsing all references of `ToText`. Don't think it's worth exposing it here.